hcx-batch-jobs/retry-batch-job/src/main/java/org/swasth/retrybatch/task/RetryBatchJob.java

```diff
@@ -0,0 +1,33 @@
+package org.swasth.retrybatch.task;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.swasth.retrybatch.functions.RetryFunction;
+
+@SpringBootApplication
+@EnableScheduling
+public class RetryBatchJob {
+
+    private final static Logger logger = LoggerFactory.getLogger(RetryBatchJob.class);
+    RetryConfig config = new RetryConfig();
+
+    public static void main(String[] args) {
+        SpringApplication.run(RetryBatchJob.class, args);
+    }
+
+    @Scheduled(fixedDelay = 5000, initialDelay = 1000)
```

Comment:
Make it configurable value

Reply:
Could not make it as a configurable value, as it must be a constant value.

Reply:
done